AntWars/Board/Ants/Ant.cs

```diff
@@ -54,7 +54,7 @@ internal Ant(Board board, Player owner, int viewRange, int maxInventory, int mov
             this.UnitsGone = 0;
             Owner = owner;
             Inventory = 0;
-            MoveRange = moveRange;
+            MoveRange = moveRange * board.diagonal;
```

Comment:
btw, sollte das nicht die kosten voll verfälschen, wenn man in der ai was kauft, wenn das dann so eine große zahl ist?